frontend/components/organisms/QuestionDetail.tsx

```diff
@@ -0,0 +1,266 @@
+                      {answer.user_id !== selectedQuestion?.user_id && (
+                        <Box
+                          minHeight={'100%'}
+                          bg={'rgba(0, 0, 0, 0.04)'}
+                          pt={'10px'}
+                          pb={'10px'}
+                          pl={'9px'}
+                          pr={'9px'}
+                        >
+                          <Avatar
+                            bg="gray.400"
+                            color="white"
+                            icon={<AiOutlineUser fontSize="22px" />}
+                            justifyContent={'center'}
+                            size={'sm'}
+                          />
+                          <Text color={PRIMARY_FONT_COLOR} fontSize={'14px'}>
+                            講師
+                          </Text>
+                        </Box>
+                      )}
+                      {answer.user_id === selectedQuestion?.user_id && (
+                        <Box pt={'10px'} pb={'10px'} pl={'9px'} pr={'9px'}>
+                          <Avatar
+                            bg="gray.400"
+                            color="white"
+                            icon={<AiOutlineUser fontSize="22px" />}
+                            justifyContent={'center'}
+                            size={'sm'}
+                          />
+                        </Box>
+                      )}
+                    </VStack>
+                    <Box
+                      pl={'15px'}
+                      pt={'10px'}
+                      pb={'10px'}
+                      className="markdown"
+                    >
+                      <ReactMarkdown>{answer.comment}</ReactMarkdown>
+                    </Box>
+                  </HStack>
+                </Card>
+              ))}
+              <Link href={`/course/${courseId}/?videoId=${videoId}`}>
+                <Button
+                  mt={'20px'}
+                  onClick={() => changeQuestionPage('QuestionList')}
+                >
+                  全ての質問へ戻る
+                </Button>
+              </Link>
+              {session &&
+                (session.user.isAdmin === true ||
+                  selectedQuestion?.user_id === session.user.id) && (
+                  <FormControl mt={'40px'}>
+                    <FormControl
+                      isInvalid={!!answerComment}
+                      mb={'20px'}
+                      bg={'white'}
+                    >
+                      <Container ml={'0px'} pb={'10px'} pl={'0px'}>
+                        <Text fontWeight={'bold'}>コメント内容</Text>
+                      </Container>
+                      <Box display={'flex'} justifyContent={'space-between'}>
+                        <Box w={'50%'} mr={'5'}>
+                          <ReactMde
+                            maxEditorHeight={250}
+                            value={answerComment}
+                            onChange={commentChange}
```

Comment:
```suggestion
                            onChange={(e) => setAnswerComment(e}
```

だと上で
commentChange
宣言しなくて良さそう